transport/src/main/java/io/netty/bootstrap/Bootstrap.java

```diff
@@ -68,11 +70,11 @@ private Bootstrap(Bootstrap bootstrap) {
      * Sets the {@link NameResolver} which will resolve the address of the unresolved named address.
      */
     @SuppressWarnings("unchecked")
-    public Bootstrap resolver(NameResolverGroup<?> resolver) {
+    public Bootstrap resolver(AddressResolverGroup<?> resolver) {
```

Comment:
That's the other way around. We switched from name resolving to address resolving, as Bootstrap indeed takes a SocketAddress.